opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -195,6 +195,15 @@ message Metric {
 // "StartTimeUnixNano" is ignored for all data points.
 message Gauge {
   repeated NumberDataPoint data_points = 1;
+
+  // sub_type provides additional type information for gauges.
+  // 
+  // When receiving from older producers of OTLP, it is safe to assume the
+  // zero-value for sub_type, GAUGE_SUBTYPE_NONE. When sending OTLP to older
+  // consumers, it is safe for them to ignore the field. In both cases, the
+  // previous behavior without sub_type is maintained until producer and
+  // consumer are aware of the field.
+  GaugeSubType sub_type = 2;
```

Comment:
Not sure if "Sub" is needed here, since we call it "GaugeType" it is a type of Gauge/sub-type of the main type.

Reply:
Removed Sub from GaugeSubType and GAUGE_SUBTYPE